CodeEditModules/Modules/AppPreferences/src/PreferencesTextEditingView.swift

```diff
@@ -13,14 +13,29 @@ public struct PreferencesTextEditingView: View {
     @AppStorage(EditorTabWidth.storageKey)
     var defaultTabWidth: Int = EditorTabWidth.default
 
+    var tabStringBinding: Binding<String> {
+        Binding<String> {
+            String(defaultTabWidth)
+        } set: {
+            defaultTabWidth = Int($0) ?? defaultTabWidth
```

Comment:
nil coalescing should probably be `EditorTabWidth.default` instead of `defaultTabWidth`

also make sure to clamp the number in a range of `0…8`. There's no such thing as negative spaces.

Reply:
This is just to avoid user input of text. If you enter text, it will be treated as if it has not been entered, and it will remain unchanged. So should use `defaultTabWidth`

The range problem fixed